res/values-pt-rBR/strings.xml

```diff
@@ -1278,7 +1278,7 @@ Esta é uma lista de coisas que é * PROIBIDO * para usar nas skins personalizad
     <string name="HERO">HERÓI</string>
     <string name="CHAMPION">CAMPEÃO</string>
     <string name="sandbox_warning" translatable="true" tools:ignore="Untranslatable">Você não pode ganhar Xp ou plasma no modo Sem Vírus (Sandbox)!</string>
-    <string name="DASH">DASH</string>
+    <string name="DASH">TRAÇO</string>
```

Comment:
Please keep it as "Dash" because many people doesn't know the translation of "Dash" and may not find the gamemode.

Reply:
Noted, thank you!